webapp/src/app/page.tsx

```diff
@@ -1,57 +1,62 @@
-import { Cache } from '@/Cache';
-import HomeDashboard from '@/components/HomeDashboard';
```

Comment:
This was the only use of HomeDashboard. Let's remove it now that we have inlined it on this page.